src/applications/personalization/dashboard/sass/user-profile.scss

```diff
@@ -8,6 +8,19 @@
   margin-bottom: 6em;
 }
 
+button.dismissible-close {
```

Comment:
Can we use design system utilities for some of these? https://design.va.gov/utilities/font-size

cc @k80bowman

Reply:
We have design system guidance on buttons here: https://design.va.gov/components/buttons
Please take a look, it looks like you are creating your own and we would like to keep button styles consistent.

Reply:
Yes, but we don't have a design system utility that floats right or makes it transparent and I don't believe you can affect the hover with the design system utilities. The screenshot below is what I am building. I'm open to better ideas, but this is what my design team made. The X is the button. You can make a dismissible button on that but it doesn't look the same so I had to make my own. I can use some of those classes, but I can't remove that class altogether as it's what is making this transparent and float right.

![Screenshot 2021-12-28 161141](https://user-images.githubusercontent.com/84803567/148457118-b7026611-c40d-469f-b470-656606eb6be0.png)

Reply:
also the design system utility has 4px not 5px for top padding so I can's use it for padding as there isn't one specific for it

Reply:
Ok I updated what I could. It still needs the float right and the hover. Let me know if you think there is anything else I can do.

Reply:
Just FYI, this component is going to be reviewed by the design system council today.  

Also just a friendly reminder that this is a prototype for usability testing and not something we're proposing to release to Veterans on production without further refinements post-usability testing.

Reply:
Oh good I'm glad to hear it will be reviewed by the design system council, that was going to be my next question. Thanks!

Reply:
@andaleliz does this need to be merged before it goes through the design system council?

Reply:
We don't need it to be merged for the design system council meeting, we have a static prototype for that. Our pilot study is Tuesday AM so ideally it'll be merged and ready for our team to review in time to make changes and prepare for that pilot. I'm totally unfamiliar with the timing of deploys and how long it takes to go to prod once merged.

Reply:
@k80bowman we intend to test it with some staging users with specific profiles before we do user testing. Preferably, we would like to test it on staging today and have monday to make any last minute changes if we see something that may need an update so Tuesday goes smoothly. That's our intention currently

Reply:
@tmitchellgcio thank you for explaining and updating what you could. After seeing the feature at the DSC meeting I understand better what it is you're doing here. Looks good.